Tests/UnitTests/Core/Geometry/CuboidVolumeBuilderTests.cpp

```diff
@@ -332,21 +333,21 @@ BOOST_AUTO_TEST_CASE(CuboidVolumeBuilderTest_confinedVolumes) {
   // Check the identified volumes
   for (unsigned int i = 0; i < stepResult.position.size(); i++) {
     if (i > 0) {
-      BOOST_TEST(stepResult.position[i].x() > 0.);
+      BOOST_CHECK_GT(stepResult.position[i].x(), 0.);
     }
     if (stepResult.position[i].x() >= 0.85 * units::_m &&
         stepResult.position[i].x() < 0.95 * units::_m) {
-      BOOST_TEST(stepResult.volume[i]->volumeName() == cvCfg2.name);
-      BOOST_TEST(stepResult.volume[i]->volumeMaterial() == nullptr);
+      BOOST_CHECK_GT(stepResult.volume[i]-, volumeName() == cvCfg2.name);
+      BOOST_CHECK_GT(stepResult.volume[i]-, volumeMaterial() == nullptr);
     } else {
       if (stepResult.position[i].x() >= 1.05 * units::_m &&
           stepResult.position[i].x() < 1.15 * units::_m) {
-        BOOST_TEST(stepResult.volume[i]->volumeName() == cvCfg1.name);
-        BOOST_TEST(stepResult.volume[i]->volumeMaterial() != nullptr);
+        BOOST_CHECK_GT(stepResult.volume[i]-, volumeName() == cvCfg1.name);
+        BOOST_CHECK_GT(stepResult.volume[i]-, volumeMaterial() != nullptr);
       } else {
         if (stepResult.position[i].x() < 2. * units::_m) {
-          BOOST_TEST(stepResult.volume[i]->volumeName() == vCfg.name);
-          BOOST_TEST(stepResult.volume[i]->volumeMaterial() == nullptr);
+          BOOST_CHECK_GT(stepResult.volume[i]-, volumeName() == vCfg.name);
+          BOOST_CHECK_GT(stepResult.volume[i]-, volumeMaterial() == nullptr);
```

Comment:
Yeah, like this. I'll fix this.

Reply:
![](https://memegenerator.net/img/instances/84836572/one-does-not-simply-parse-c-code-using-a-regex.jpg)

Reply:
I tend to follow a semi-automated approach for this kind of work: configure a regex-based search and replace in my text editor, but manually go through every occurence before accepting the replacement to make sure that there are no regex false positives (interactively improving my regex if need be).